core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -5551,6 +5560,8 @@ private static String javadocLink(String fullyQualifiedClassname) {
     public static final String FUSE_DEBUG_ENABLED = "alluxio.fuse.debug.enabled";
     public static final String FUSE_FS_NAME = "alluxio.fuse.fs.name";
     public static final String FUSE_JNIFUSE_ENABLED = "alluxio.fuse.jnifuse.enabled";
+    public static final String FUSE_SHARED_GRPC_DATA_READER
```

Comment:
I think it's specific to FUSE use case, not sure if it's able to work on other cases.
The `bufferCachingDataReader` is more general, but the `SharedDataReader` can be only used for fuse since it is based on the assumption
```
 * It follows GrpcDataReader protocol and takes strong assumption:
 * Parallel read to the same file happens on the same time, so that read request is
 * serialized by kernel
```
it relies on one thread do seek() and then readChunk()  and then the other thread do seek() -> readChunk()s.

Reply:
Maybe we can change the current name `SharedDataReader` to `FuseSharedDataReader` to be more clear?